### PR TITLE
Handle processing type now that it is a data field in JSON

### PR DESCRIPTION
**Pull request by dpattermann-usgs**
Companion PR to https://github.com/USGS-CIDA/AQCU/pull/451

### CODE REVIEW

File: R/uvhydrograph-render.R [hunk 1]
```diff
@@ -36,10 +36,12 @@ createPrimaryPlot <- function(data, month){
   if(anyDataExist(primaryData)){
 
     primaryInfo <- parseUVSupplemental(data, "primary", primaryData)
+    plotEndDate <- tail(primaryInfo$plotDates,1) + hours(23) + minutes(45)
```
Comment (mhines-usgs) [inline R/uvhydrograph-render.R:39]: @dpattermann-usgs is this the suggestion you have on the premature end of time range issue I'll get to in https://internal.cida.usgs.gov/jira/browse/AQCU-670

Comment (dpattermann-usgs) [inline R/uvhydrograph-render.R:39]: @mhines-usgs Yeah, this is the way I did it for UV Hydro. It feels kind of hacky to do it this way? If you can think of a better way, you should definitely tell me.

File: R/correctionsataglance-data.R [hunk 2]
```diff
@@ -6,10 +6,14 @@ parseCorrectionsData <- function(data){
   apprData <- formatDataList(data$primarySeries$approvals, 'APPROVALS', datesRange = dateData$dateSeq) #top bar = primary series approvals
   fieldVisitData <- list(startDates = formatDates(data$fieldVisits$startTime),
                          dataNum = length(data$fieldVisits$startTime)) #points on top bar = field visits
-  PREData <- formatDataList(data$corrections, "PRE_PROCESSING") #lane one = pre-processing
-  NORMALData <- formatDataList(data$corrections, "NORMAL") #lane two = normal
-  POSTData <- formatDataList(data$corrections, "POST_PROCESSING") #lane three = post-processing
-
+  
+  PREData <- data$corrections$PreProcessing #lane one = pre-processing
```
Comment (thongsav-usgs) [inline R/correctionsataglance-data.R:10]: Capitalization might change on the data structure fields.

Comment (lindsayplatt) [inline R/correctionsataglance-data.R:10]: :hankey:

Comment (dpattermann-usgs) [inline R/correctionsataglance-data.R:10]: What do you mean?

Comment (lindsayplatt) [inline R/correctionsataglance-data.R:10]: I interpreted this as `data$corrections$PreProcessing` might now be `data$corrections$preprocessing` or something similar

Comment (dpattermann-usgs) [inline R/correctionsataglance-data.R:10]: Okay, done.

**Pull request merged**